junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -93,7 +93,7 @@ void notifyFinished() {
         switch (state) {
         case INIT:
             notifyPrepared();
-            break;
+//            break;
```

Comment:
I see, I understand now.  There, was probably an ambiguity in this original code, specifically where script execution has finished.  Specifically, after the prepare/prepared but before started script execution could complete if it was all server channels.  The case here is probably covering that with the notifyPrepared() being called in notifyFinished.

It was unclear to me what the best behavior should be, but we should specify either 

1) the FINISHED event can only come after a START/STARTED,
or 
2) whether it can come after prepared (in case of server channels).

In my PR and what we had originally specified is 1.  But now after thinking about it, perhaps 2 is better because it allows early termination of completed or partially completed and then errored scripts.  Thoughts?